src/config/errors.ts

```diff
@@ -37,6 +37,16 @@ export class AccessRestrictedToPublicServiceEmailError extends Error {
   }
 }
 
+export class DomainRestrictedError extends Error {
+  constructor(
+    public organizationId: number,
+    options?: ErrorOptions,
+  ) {
+    super("", options);
```

Comment:
Suggestion: I'm not sure why the error you introduced is affected by the options passed to the parent, while mine isn't. Could you take a pass at making this consistent?